src/package-manager.ts

```diff
@@ -0,0 +1,32 @@
+import { existsSync } from "node:fs"
+import { select, confirm } from "@inquirer/prompts"
+import { PackageManager, lockFiles, packageManagers } from "./constants"
+
+export default async function choosePackageManager(): Promise<PackageManager> {
+	const pm = findPackageManager()
```

Comment:
let's use `packageManager` for clarity, pm confuses me a bit :)
```suggestion
	const packageManager = findPackageManager()
```